### PR TITLE
feat: Changed link implementation using path and gradient

### PR DESCRIPTION
**Pull request by Adityapnn811**
Okay I finally figure out how the SVG path works, sorry it took long enough. I changed the way gradient is implemented because currently it's the most efficient way to apply strokes and change it according to the nodes' position.

### CODE REVIEW

**Comment by christojeffrey** [comment]
image


looks good i guess

File: svelte/src/routes/v2/scripts/draw/tick.ts [hunk 1]
```diff
@@ -1,63 +1,35 @@
...
-		.attr('y1', (d: any) => {
-			// change to global coordinates.
-			let result = d.source.y;
-
-			let temp = d.source;
-			while (temp.parent) {
-				result += temp.parent.y;
-				temp = temp.parent;
-			}
+	linkElements.attr('d', function (this: SVGPathElement, d: any) {
```
Comment (christojeffrey) [inline svelte/src/routes/v2/scripts/draw/tick.ts:9]: this part will conflict with my next PR. I think I am gonna need your help to merge them. I hope that's okay with you

**Review by christojeffrey — APPROVED** [review]
good. minimal changes

File: svelte/src/routes/v2/scripts/draw/tick.ts (hunk 1 above)
Comment (Adityapnn811) [reply svelte/src/routes/v2/scripts/draw/tick.ts:9]: Okay no problem

**Pull request merged**